src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,77 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event.
+ *
+ * @author Pearlynn
+ */
+
+public class Event extends Task {
+
+    /**
+     * The start date/time of the event.
+     */
+    protected LocalDateTime from;
+
+    /**
+     * The end date/time of the event.
+     */
```

Comment:
I noticed the same issue in several other places too regarding the coding standard for variable commenting